baseplate/thrift/baseplate.thrift

```diff
@@ -87,11 +87,30 @@ struct OriginService {
     1: string name
 }
 
+/** Geolocation data from a request to our services that we want to
+propagate between services.
+
+This model is a component of the "Edge-Request" header.  You should not need to
+interact with this model directly, but rather through the EdgeRequestContext
+interface provided by baseplate.
+
+*/
+struct Geolocation {
+    /** The country code of the requesting client.
+    */
+    1: CountryCode country_code
+}
+
 /**A raw authentication token as returned by the authentication service.
 
 */
 typedef string AuthenticationToken
 
+/** A two-character ISO 3166-1 country code
+
+*/
+typedef string CountryCode
```

Comment:
Should the typedef come before `Geolocation`?

Reply:
How about we move all typedefs to after the namespace declarations?